assets/prefabs/structures/Smith.prefab

```diff
@@ -0,0 +1,70 @@
+            { "blockType": "CoreBlocks:DoorTop.BACK", "region": { "min": [-2, 2, 1], "size": [2, 1, 1]}},
+            { "blockType": "CoreBlocks:Glass", "region": { "min": [-4, 2, 3], "size": [1, 1, 2]}},
+            { "blockType": "CoreBlocks:Glass", "region": { "min": [4, 2, 3], "size": [1, 1, 2]}},
+            { "blockType": "CoreBlocks:Glass", "region": { "min": [2, 2, 6], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.BACK", "region": { "min": [-2, 6, 4], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.BACK", "region": { "min": [-3, 5, 5], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.BACK", "region": { "min": [-4, 4, 6], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.FRONT", "region": { "min": [4, 4, 1], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.FRONT", "region": { "min": [3, 5, 2], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.FRONT", "region": { "min": [2, 6, 3], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.LEFT", "region": { "min": [-4, 4, 1], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.LEFT", "region": { "min": [-3, 5, 2], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.LEFT", "region": { "min": [-2, 6, 3], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.RIGHT", "region": { "min": [2, 6, 4], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.RIGHT", "region": { "min": [3, 5, 5], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.RIGHT", "region": { "min": [4, 4, 6], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownRoof.BACK", "region": { "min": [-1, 6, 4], "size": [3, 1, 1]}},
+            { "blockType": "HjerlHede:brownRoof.BACK", "region": { "min": [-2, 5, 5], "size": [5, 1, 1]}},
+            { "blockType": "HjerlHede:brownRoof.BACK", "region": { "min": [-3, 4, 6], "size": [7, 1, 1]}},
+            { "blockType": "HjerlHede:brownRoof.FRONT", "region": { "min": [-3, 4, 1], "size": [7, 1, 1]}},
+            { "blockType": "HjerlHede:brownRoof.FRONT", "region": { "min": [-2, 5, 2], "size": [5, 1, 1]}},
+            { "blockType": "HjerlHede:brownRoof.FRONT", "region": { "min": [-1, 6, 3], "size": [3, 1, 1]}},
+            { "blockType": "HjerlHede:brownRoof.LEFT", "region": { "min": [-4, 4, 2], "size": [1, 1, 4]}},
+            { "blockType": "HjerlHede:brownRoof.LEFT", "region": { "min": [-3, 5, 3], "size": [1, 1, 2]}},
+            { "blockType": "HjerlHede:brownRoof.RIGHT", "region": { "min": [4, 4, 2], "size": [1, 1, 4]}},
+            { "blockType": "HjerlHede:brownRoof.RIGHT", "region": { "min": [3, 5, 3], "size": [1, 1, 2]}},
+            { "blockType": "HjerlHede:sorringladendiagonal", "region": { "min": [-3, 1, 1], "size": [1, 1, 1]}},
```

Comment:
I'm wondering how/why you use both `HjerlHede:sorringladendiagonal` and `StructuralResources:TimberFrameDiagonal`. Is this by mistake or on purpose? 

I would appreciate if you could use resources from StructuralResources where possible, i.e., replace the occurrences of the `sorringladen` blocks with the equivalents from StructuralResources.

Reply:
`HjerlHede:sorringladendiagonal` and `StructuralResources:TimberFrameDiagonal` have different directions one of them is to the left and other is to the right. I guess you didn't noticed but I changed `HjerlHede:sorringladenplain` to `StructuralResources:TimberFramePlain` and `HjerlHede:sorringladenmiddle` to `StructuralResources:TimberFrameVertical`.